adding_species_manuscript.md

```diff
@@ -54,51 +61,63 @@ into the stdpopsim catalog in order to make it available
-Correspondingly, methods for inferring demographic history and natural selection from such data are flourishing @Beichman2018.
-Past methods development has often focused on model organisms such as *Drosophila* or, piggybacking on medical genomics, humans.
-But new methods are being developed, and old methods enhanced, to model population characteristics that are key to many organisms but less important in these traditional models, such as inbreeding @Blischak2020 or skewed offspring distributions @Montano2016.
-(RNG: Any models include these features in the catalog? MEL: C. reinhardtii? popsim-consortium/stdpopsim#863; Demes can do selfing: popsim-consortium/stdpopsim#857)
-Models inferred from these tools are an important community resource, but reuse of such models can be arduous and error-prone @Ragsdale 2020.
+Dramatic reductions in sequencing costs are enabling unprecedented genomic data to be generated for a huge variety of species (@Ellegren2014). Ongoing efforts to systematically sequence a wide variety of species (eg. Vertebrate Genomes Project (@Rhie2021), 10KP (10,000 Plants) Genome Project (@Cheng2018)) are in turn facilitating enormous increases in population-level genomic data for new model and non-model species.
+Correspondingly, methods for inferring demographic history and natural selection from such data are flourishing (@Beichman2018).
+Past methods development has typically focused on model organisms such as *Drosophila* or, piggybacking on medical genomics, humans.
+But new methods are being developed, and old methods enhanced, to model population characteristics that are key to many organisms but less important in these traditional models, such as inbreeding (@Blischak2020), skewed offspring distributions (@Montano2016), or selfing (eg. as implemented in Demes (@Gower2022)).
+Models inferred from these tools are an important community resource, but reuse of such models can be arduous and error-prone (@Ragsdale2020).
 
 Simulations from population genomic models have many uses, both for methods development and empirical research.
-They provide training data for inference methods based on machine learning @Schrider2018 or Approximate Bayesian Computation @Csillery2010.
-They can also serve as baselines for further analyses; for example, models incorporating demographic history serve as null models in selection analyses @Hsieh2016a.
-(RNG: This self-cite might not be the best reference.)
-Population genomic simulations can also help guide conservation decisions for threatened species @Teixeira2021.
+They provide training data for inference methods based on machine learning (@Schrider2018) or Approximate Bayesian Computation (@Csillery2010).
+They can also serve as baselines for further analyses; for example, models incorporating demographic history serve as null models in selection analyses (@Hsieh2016a).
+Population genomic simulations can also help guide conservation decisions for threatened species (@Teixeira2021).
```

Comment:
Maybe add something like: "**More recently**, population genomic simulation has been used to guide conservation ....